examples/js/loaders/GLTFLoader.js

```diff
@@ -615,29 +601,18 @@ THREE.GLTFLoader = ( function () {
 					'material.specularColor = specularFactor.rgb;',
 				].join( '\n' );
 
-				var fragmentShader = shader.fragmentShader
```

Comment:
this is all actually broken but the idea is that the loader would be decoupled from this